.github/workflows/build.yml

```diff
@@ -1,5 +1,9 @@
+name: Build Kernel
 on:
   push:
+    branches:
+      - main
```

Comment:
why limit what branches to run build on?

Reply:
hm, not sure why I did this, I think it was a misconception. thanks for pointing this out.